xml/System.Data/DataReaderExtensions.xml

```diff
@@ -751,12 +882,31 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <param name="name">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="reader">The <see cref="T:System.Data.Common.DbDataReader" /> to get the column value from.</param>
+        <param name="name">The name of the column.</param>
+        <param name="cancellationToken">A System.Threading.CancellationToken to cancel the asynchronous operation.</param>
+        <summary>Asynchronously gets a value that indicates whether the column contains non-existent or missing values.</summary>
+        <returns>
+          <see langword="true" /> if the specified column value is equivalent to <see langword="DBNull" /> otherwise <see langword="false" />.</returns>
```

Comment:
Actually returns t Task<bool\>. Standard way to say this?

Reply:
The [System.IO.FileStream.ReadAsync](https://docs.microsoft.com/en-us/dotnet/api/system.io.filestream.readasync?view=netcore-3.0#System_IO_FileStream_ReadAsync_System_Memory_System_Byte__System_Threading_CancellationToken_ 
) method is a good example you can imitate:

> A task that represents the asynchronous read operation. The value of the TResult parameter contains the total number of bytes read into the buffer. The result value can be less than the number of bytes requested if the number of bytes currently available is less than the requested number, or it can be 0 (zero) if the end of the stream has been reached.